filesystem/util.go

```diff
@@ -58,12 +67,21 @@ func IsMounted(device, target string) (bool, error) {
 			continue
 		}
 
+		// If the filesystem is nfs and its connection is broken, EvalSymlinks will be stuck.
+		// So it should be in before calling EvalSymlinks.
+		ok, err := isSameDevice(device, fields[0])
```

Comment:
Is there any possibility to break idempotency like continuous mounting the same filesystem for each reconciliation loop?

Reply:
I think no.
Could you tell me a concrete case?